joomla/joomla-cms
# Update composer and remove dummy random_compat

### Summary of Changes

Remove the dummy random_compat https://github.com/paragonie/random_compat#version-99999

Package operations: 0 installs, 19 updates, 0 removals
  - Updating psr/link (1.0.0 => 1.1.1): Loading from cache
  - Updating fig/link-util (1.1.2 => 1.2.0): Loading from cache
  - Updating laminas/laminas-zendframework-bridge (1.1.1 => 1.2.0): Loading from cache
  - Updating laminas/laminas-diactoros (2.4.1 => 2.6.0): Loading from cache
  - Updating symfony/polyfill-php80 (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/polyfill-mbstring (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/var-dumper (v5.2.6 => v5.2.8): Loading from cache
  - Updating symfony/polyfill-php72 (v1.22.1 => v1.23.0): Loading from cache
  - Updating psr/log (1.1.3 => 1.1.4): Loading from cache
  - Updating symfony/error-handler (v5.2.6 => v5.2.8): Loading from cache
  - Updating symfony/polyfill-php73 (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/polyfill-ctype (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/yaml (v5.2.5 => v5.2.9): Loading from cache
  - Updating symfony/polyfill-intl-normalizer (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/polyfill-intl-grapheme (v1.22.1 => v1.23.0): Loading from cache
  - Updating symfony/string (v5.2.6 => v5.2.8): Loading from cache
  - Updating symfony/polyfill-iconv (v1.22.1 => v1.23.0): Loading from cache
  - Updating fgrosse/phpasn1 (v2.2.0 => v2.3.0): Loading from cache
  - Updating symfony/console (v5.2.6 => v5.2.8): Loading from cache

### Testing Instructions

install joomla
apply this patch
run composer install --no-dev (or use the generated package)
Make sure joomla still loads.

### Actual result BEFORE applying this Pull Request

There is an empty random_compat package while the 9.99.9 version is empty and does nothing anyway.

### Expected result AFTER applying this Pull Request

That package is gone and the other listed packages are updated.

### Documentation Changes Required

none
